src/network.rs

```diff
@@ -0,0 +1,88 @@
+use errors::*;
+use std::path::{Path,PathBuf};
+use std::fs::{remove_file, rename};
+
+use args::Args;
+use fs;
+
+const BOOTPART_CONNECTIONS_DIR: &str = "system-connections";
+const NM_CONNECTIONS_DIR: &str = "/etc/NetworkManager/system-connections/";
+
+pub fn configure(args: &Args) -> Result<()> {
+    mirror_connections(args, BOOTPART_CONNECTIONS_DIR, NM_CONNECTIONS_DIR)?;
+    Ok(())
+}
+
+fn no_comments(s: &str) -> String {
+    let mut ret = String::new();
+    for line in s.lines() {
+        if ! line.starts_with("#") {
+            ret = ret + line + "\n";
+        }
+    }
+    ret
+}
+
+pub fn mirror_connections(args: &Args, bootpart_connections_dir: &str, nm_connections_dir: &str) -> Result<()> {
+
+    const IGNORE_EXT: &str = "ignore";
```

Comment:
I prefer having constants defined on the top of the module instead of inside functions.